File: src/main/java/com/stemmildt/reactiveproducts/ReactiveProductsFeatureClassProvider.java
package com.stemmildt.reactiveproducts;

import org.springframework.stereotype.Component;
import org.togglz.core.Feature;

import com.breuninger.boot.togglz.FeatureClassProvider;

@Component
public class ReactiveProductsFeatureClassProvider implements FeatureClassProvider {

  @Override
  public Class<? extends Feature> getFeatureClass() {
    return Features.class;
  }
}

File: src/main/kotlin/com/stemmildt/reactiveproducts/ReactiveProductsFeatureClassProvider.kt
package com.stemmildt.reactiveproducts;

import com.breuninger.boot.togglz.FeatureClassProvider
import org.springframework.stereotype.Component
import org.togglz.core.Feature

@Component
class ReactiveProductsFeatureClassProvider : FeatureClassProvider {

  override fun getFeatureClass(): Class<out Feature> = Features::class.java
}
